doc/background/components/components_overview.md

```diff
@@ -0,0 +1,231 @@
+# Component overview
+
+Panel provides a wide range of components for easily composing panels, apps, and dashboards both in the notebook and as standalone apps. The components can be broken down into three broad classes of objects:
+
+* ``Pane`` objects allow wrapping external viewable items like Bokeh, Plotly, Vega, or HoloViews plots, so they can be embedded in a panel.
+* ``Widget`` objects provide controls that can trigger Python or JavaScript events.
+* ``Panel`` layout objects allow combining plots into a ``Row``, ``Column``, ``Tabs`` or a ``Grid``.
+
+All objects share an API that makes it easy to [customize their layout behavior and visual appearance](./Customization.md), [link](./Links.md) and [display and export](./Display_and_Export.rst) them. To display any panel objects in a notebook environment ensure you load the extension first:
```

Comment:
These cross-references are invalid and should link to how-to sections OR Background stubs.

Reply:
done